swap/swap.go

```diff
@@ -157,7 +175,7 @@ func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 	// It is the peer with a negative balance who sends a cheque, thus we check
 	// that the balance is *below* the threshold
 	if swapPeer.getBalance() <= -s.paymentThreshold {
-		log.Warn("balance for peer went over the payment threshold, sending cheque", "peer", peer.ID().String(), "payment threshold", s.paymentThreshold)
+		s.logger.Warn("balance for peer went over the payment threshold, sending cheque", "peer", peer.ID().String(), "payment threshold", s.paymentThreshold)
```

Comment:
Not related to this PR, but it strikes me that here only the debitor will get this warning. The creditor (the one who is owned to) does never ever have a warning saying "Hey, one of your peers got into debt with you". It only gets notified when a node sends the actual cheque already.

Just thinking out loud, let's think if we need this at all (maybe we already talked about this and I forgot).

Reply:
Good one. I would like to see such a log as well. We never discussed it, as far as I know.

Reply:
Good point.
Would this be a separate PR?

Reply:
Yes, let's create an issue and do a separate PR for this, and discuss it maybe in the next sync meeting.